### PR TITLE
Noorin Sakhi- I have added my profile

### LINKED ISSUE #604: Noorin Sakhi
my profile

### PR DESCRIPTION
**Pull request by NoorinSakhi**
Resolve #604 .

### CODE REVIEW

**Review by bishalr0y — COMMENTED** [review]
Hello @NoorinSakhi, kindly add your profile at the end of the file.

**Comment by NoorinSakhi** [comment]
Ok, I will. Thanks

**Comment by NoorinSakhi** [comment]
I pushed the changes

**Review by bishalr0y — APPROVED** [review]

**Comment by HareerSakhi** [comment]
@kaiwalyakoparkar I think that this PR can be merged (:

File: data.json [hunk 1]
```diff
@@ -1144,19 +1144,24 @@
...
-
   },
   {
     "name": "Rohit Yadav",
     "githubUsername": "rohitturbot",
     "imageUrl": "https://user-images.githubusercontent.com/40739903/126065686-40212b44-01a8-4c5f-b38a-de22ca76cc9c.png",
     "issueNumber": 602
   },
+  {
+
```
Comment (kaiwalyakoparkar) [inline data.json:1155]: You can remove this line as it breaks the code style :) Just remove that empty line and the PR is good to go

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

File: data.json (hunk 1 above)
Comment (kaiwalyakoparkar) [reply data.json:1155]: Line no `1155`

Comment (NoorinSakhi) [reply data.json:1155]: Ok, I will now.

Comment (NoorinSakhi) [reply data.json:1155]: I can't see where to remove line `1155`: 
image

Comment (NoorinSakhi) [reply data.json]: Just found and did it. Thank you.

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket:  Thank you for your collaboration and contribution :tada:  Thank you @HareerSakhi & @bishalr0y for reviewing the PR

**Pull request merged**